src/lib/margaux.js

```diff
@@ -397,11 +267,144 @@ export function forceCharset (chrome: Chrome, callback: Callback) {
             resolve(null)
           })
         })
-      })).then(function (results) {
-        callback(null, results)
-      }).catch(function (err) {
-        return callback(new Error(err))
       })
+      ).then(results => callback(null, results)
+      ).catch(err => callback(new Error(err)))
+    })
+  })
+}
+
+export function close (client: CDP, cb: (err: ?Error, res: string) => void) {
+  const {host, port, tab} = client
+  CDP.Close({
+    host: host,
+    port: port,
+    id: tab.id
```

Comment:
https://github.com/cyrus-and/chrome-remote-interface/commit/4ce3560affac6837054f7a8e73a3760ce1475ab5